### PR TITLE
Update CVE-2022-21504

### PR DESCRIPTION
**Pull request by bsitu**
On branch cna/Oracle/CVE-2022-21504
 Changes to be committed:
	modified:   2022/21xxx/CVE-2022-21504.json

### CODE REVIEW

File: 2022/21xxx/CVE-2022-21504.json [hunk 1]
```diff
@@ -1,17 +1,76 @@
+
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: ('Expecting property name enclosed in double quotes: line 34 column 25 (char 1180)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/21xxx/CVE-2022-21504.json [hunk 2]
```diff
@@ -1,18 +1,76 @@
...
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: ('Expecting value: line 33 column 25 (char 1171)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/21xxx/CVE-2022-21504.json [hunk 3]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: ('Expecting value: line 32 column 25 (char 1160)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/21xxx/CVE-2022-21504.json (hunk 3 above)
Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/21xxx/CVE-2022-21504.json:1]: ('Expecting property name enclosed in double quotes: line 72 column 13 (char 3110)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by bsitu** [comment]
The last comment shown above is "This branch has no conflicts with the base branch". Is there something else preventing the merge? Thanks.

**Comment by bsitu** [comment]
Thank you!

**Pull request merged**